lib/xboxkrnl/xboxkrnl.h

```diff
@@ -3,7 +3,7 @@
 // SPDX-FileCopyrightText: 2017-2023 Stefan Schmidt
 // SPDX-FileCopyrightText: 2018-2021 Jannik Vogel
 // SPDX-FileCopyrightText: 2018 Sean Koppenhafer
-// SPDX-FileCopyrightText: 2022 Erik Abair
+// SPDX-FileCopyrightText: 2022-2025 Erik Abair
```

Comment:
nit: I'm not sure if we also use separate years elsewhere or if mixing of ranges and separate is common (e.g: `x-y,z,u-v`), but *if* we do, I think `2022,2025` would be more faithful in this instance.

Reply:
I'm fine just removing myself entirely, I would hope the copyright could be assigned to the project without the need to have individual names and worrying about assigning timestamps to contributions and whether they should be comma separated if intervening years had no commits, etc...

Reply:
The SPDX spec isn't very precise regarding copyright texts, so mixing is probably fine. Personally I don't really care whether it's 2022-2025 or 2022,2025 here, but please don't remove your name.

> copyright could be assigned to the project

Oh we don't do that. Giving away your copyright is unnecessary imho and doesn't even exist as a legal possibility in some countries, it's why some projects make you sign huge contributor license agreements that cover all kinds of strange cases. I stay as far away from such stuff as possible.

Reply:
> nit: I'm not sure if we also use separate years elsewhere or if mixing of ranges and separate is common

https://github.com/search?q=repo%3AXboxDev%2Fnxdk+SPDX-FileCopyrightText&type=code

According to code search (see link above), they all contain dashes for an initial year to the "last time touched" current year. Although, it's kind of a moot point to update the "last time touched" current year when we have git blame to inspect who made the latest changes. General companies did drop the range of updated year anyway.

If it were up to me, I would just use the initial year only. That's my personal opinion.

Reply:
I'm happy with that last suggestion. I've restored the original state w/ just the original contribution date. 

It'd be really good to either clean out the dashes or provide some contributor instructions that indicate that people shouldn't bother extending ranges if they commit over years so it's clear what is expected.